editor/editor_themes.cpp

```diff
@@ -702,6 +702,19 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_color("icon_focus_color", "Button", icon_focus_color);
 	theme->set_color("icon_pressed_color", "Button", icon_pressed_color);
 
+	// Variation for Editor Log filter buttons
+	theme->set_type_variation("editor_log_filter_button", "Button");
+	// When pressed, don't tint the icons with the accent color, just leave them normal.
+	theme->set_color("icon_pressed_color", "editor_log_filter_button", Color::named("WHITE"));
```

Comment:
This is already default color. See `icon_normal_color`.

Reply:
This is overriding `icon_pressed_color`, not normal color. Pressed color tints it blue currently. See image in OP. Although you are right, I can just use `icon_normal_color`.